nsxt/resource_nsxt_vpc_subnet.go

```diff
@@ -209,13 +213,13 @@ func resourceNsxtVpcSubnet() *schema.Resource {
 		Update: resourceNsxtVpcSubnetUpdate,
 		Delete: resourceNsxtVpcSubnetDelete,
 		Importer: &schema.ResourceImporter{
-			State: nsxtParentPathResourceImporter,
+			State: nsxtVPCPathResourceImporter,
```

Comment:
If this has already been fixed in the generator maybe we can try and update the generated commit so we don't have to manually change the importer?

Reply:
If I understand correctly, `nsxtParentPathResourceImporter` is useful for objects which have sub-objects of a level VPC obj, e.g `VpcSubnetPort` which is nested under `VpcSubnet` and therefore has  a parent object path.
IDK if we can conclude automatically from the spec if this is the case or not. @annakhm do you know?

Reply:
This fix is already present in generator - based on whether `parent_path` is specified or not, the correct importer should be chosen